scalyr_agent/scalyr_monitor.py

```diff
@@ -270,24 +271,32 @@ def run(self):
         # noinspection PyBroadException
         try:
             while not self._is_thread_stopped():
+                sample_interval = self._sample_interval_secs
+                if self._adjust_sleep_by_gather_time:
+                    random_jitter = 0
+                elif self._global_config.global_monitor_sample_interval_enable_jitter:
+                    # min jitter is 1/10 of the sample interval and max is 2/10
```

Comment:
Eh, I might do 8/10 for the max.  Delaying by one sample period should not be a big deal.

Reply:
Yeah, I think this should be reasonable, especially for "normal" sample intervals (think 30 or 60 seconds).

If someone intentionally sets it to something large (e.g. 15 or 30 minutes), it may be a bigger issue. We could add some kind of upper bound for it (e.g. 1 minute), but I also don't think many people actually use that long intervals.

Reply:
Good idea about capping to a maximum size.